src/components/Footer/Footer.tsx

```diff
@@ -98,7 +98,10 @@ export const Footer: React.FC<Props> = ({
       </div>
       <div className="usa-footer__secondary-section bg-primary-darker text-white font-sans-2xs padding-x-4">
         <div className="grid-row">
-          <p className="tablet:grid-col-9 mobile-text-center">{`Email ${emailAddress} for help or feedback.`}</p>
+          <p className="tablet:grid-col-9 mobile-text-center">
+            Email {<b className="text-underline">{emailAddress}</b>} for help or
```

Comment:
Should we make this a mailto link?
```suggestion
            Email <a href=`mailto:${emailAddress}`><b className="text-underline">{emailAddress}</b></a> for help or
```